.github/workflows/CI.yml

```diff
@@ -0,0 +1,74 @@
+name: CI
+# Run on master, any tag or any pull request
+on:
+  push:
+    branches:
+      - master
+      - staging
+      - trying
```

Comment:
Probably don't want to leave this branch here. ALso @fchorney can you review this file for other things that need updating?

Reply:
I think `trying` is for bors. Will that be used for this?

Reply:
Yes `bors` uses both `staging` and `trying`.